user_tools/src/spark_rapids_tools/tools/qualx/preprocess.py

```diff
@@ -1074,10 +1077,14 @@ def load_qtool_execs(qtool_execs: List[str]) -> Optional[pd.DataFrame]:
     if qtool_execs:
         exec_info = pd.concat([pd.read_csv(f) for f in qtool_execs])
         node_level_supp = exec_info.copy()
-        node_level_supp['Exec Is Supported'] = node_level_supp[
-            'Exec Is Supported'
-        ] | node_level_supp['Exec Name'].apply(
-            lambda x: any([x.startswith(nm) for nm in unsupported_overrides])
+        node_level_supp['Exec Is Supported'] = (
+            node_level_supp['Exec Is Supported']
+            | node_level_supp['Exec Name'].apply(
+                lambda x: any([x.startswith(nm) for nm in unsupported_overrides])
+            )
+            | node_level_supp['Exec Name'].apply(
+                lambda x: x.startswith('WholeStageCodegen')
+            )
```

Comment:
I thought that change was rolled back..Just double checking.

Reply:
Per @eordentlich's commit to internal repo, this is a workaround for#860.